tests/verifyAuroraRemoteInstance_test.js

```diff
@@ -0,0 +1,120 @@
+const assert = require('assert');
+const { remoteInstancesHelper, pmmInventoryPage } = inject();
+
+Feature('Monitoring Aurora instances');
+
+const instances = new DataTable(['service_name', 'password', 'instance_id', 'cluster_name']);
+const metric = 'mysql_global_status_max_used_connections';
+
+instances.add([
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora2.address,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora2.password,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora2.instance_id,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora2.cluster_name,
+]);
+instances.add([
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora3.address,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora3.password,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora3.instance_id,
+  remoteInstancesHelper.remote_instance.aws.aurora.aurora3.cluster_name,
+]);
+
+Before(async ({ I }) => {
+  await I.Authorize();
+});
+
+Data(instances).Scenario('PMM-T1295 Verify adding Aurora remote instance @instances', async ({ I, addInstanceAPI, current }) => {
+  const { service_name, password, instance_id, cluster_name } = current;
+
+  const details = {
+    add_node: {
+      node_name: service_name,
+      node_type: 'REMOTE_NODE',
+    },
+    aws_access_key: remoteInstancesHelper.remote_instance.aws.aurora.aws_access_key,
+    aws_secret_key: remoteInstancesHelper.remote_instance.aws.aurora.aws_secret_key,
+    address: service_name,
+    service_name: instance_id,
+    port: remoteInstancesHelper.remote_instance.aws.aurora.port,
+    username: remoteInstancesHelper.remote_instance.aws.aurora.username,
+    password: password,
+    instance_id: instance_id,
+    cluster: cluster_name,
+  };
+
+  await addInstanceAPI.addRDS(details.service_name, details);
+
+  I.amOnPage(pmmInventoryPage.url);
+  pmmInventoryPage.verifyRemoteServiceIsDisplayed(details.service_name);
+  await pmmInventoryPage.verifyAgentHasStatusRunning(details.service_name);
+});
+
+Data(instances)
+  .Scenario('PMM-T1295 Verify Aurora instance metrics @instances', async ({ I, current, grafanaAPI }) => {
+    const { instance_id } = current;
+
+    // Waiting for metrics to start hitting for remotely added services
+    I.wait(10);
+
+    // verify metric for client container node instance
+    const response = await grafanaAPI.checkMetricExist(metric, {
```

Comment:
any reason why you are not checking for RDS related metrics, Aurora related metrics?

Reply:
We don't have aurora metrics for Aurora3 due to a bug. So I left the test which checks 'mysql_global_status_max_used_connections' for both Aurora2 and 3 and added another test which checks 'mysql_global_status_aurora_total_op_memory' only for Aurora2 until the bug is fixed.